index.html

```diff
@@ -0,0 +1,219 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Addisalem seifu Portofolio</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/e1cc58c746.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    
+    <section class="front_page">
+        <div class="nav__wrapper">
+            <nav class="nav_bar">
+                <h2 class="logo">My Logo</h2>
+                <ul class="menue_bar">
+                    <li class="bar1"></li>
+                    <li class="bar1"></li>
+                    <li class="bar1"></li>
+                </ul>
+                <ul class="menue_bar_desktop">
+                    <li class="nav_list"><a href="#" class="nav_link">Portofolio</a></li>
+                    <li class="nav_list"><a href="#" class="nav_link">About</a></li>
+                    <li class="nav_list"><a href="#" class="nav_link">Contact</a></li>
+                </ul>
+            </nav>
+        </div>
+        <div class="main__wrapper">
```

Comment:
Please use background images for the Headline section as decoration from the design page.